src/commands/flutter-symbols/README.md

```diff
@@ -0,0 +1,53 @@
+## Overview
+
+To deobfuscate and symbolicate errors and crashes, upload your Flutter Symbols, iOS dSYMs, and Android mapping Files to Datadog
+
+## Setup
+
+You need to have `DATADOG_API_KEY` in your environment.
+
+```bash
+# Environment setup
+export DATADOG_API_KEY="<API KEY>"
+```
+
+You can configure the tool to use Datadog EU by defining the `DATADOG_SITE` environment variable as `datadoghq.eu`. By default, the requests are sent to Datadog US.
+
+To make these variables available, Datadog recommends setting them in an encrypted `datadog-ci.json` file at the root of your project:
+
+```json
+{
+  "apiKey": "<DATADOG_API_KEY>",
+  "datadogSite": "<DATADOG_SITE>"
+}
+```
+
+To override the full URL for the intake endpoint, define the `DATADOG_SOURCEMAP_INTAKE_URL` environment variable.
+
+## Commands
+
+### `upload`
+
+This command uploads your Dart symbol files, iOS dSYMs, and Android Proguard mapping files to Datadog in order to deobfuscate and symbolicate your application's stack traces.
+
+After running `flutter build --split-debug-info=./debug-info --obfuscate`, you can upload all files to Datadog by running this command: 
+
+```bash
+datadog-ci flutter-symbols upload --dart-symbols-location ./debug-info --service-name com.companyname.application --ios-dyms --android-mapping
+```
+
+| Parameter | Condition | Description |
+|-----------|-----------|-------------|
+| `--service-name` | Required | Set as the service name you are uploading files for. Datadog uses this service name to find corresponding files based on the `service` options set during configuration of the the RUM Flutter SDK.<br>By default, the RUM Flutter SDK uses your application's bundle identifier as the service. |
+| `--flavor` | Optional | The build flavor that was built. Defaults to `release` |
+| `--dart-symbols-location`  | Optional  | The locaiton of your Dart symbol files. This should be the same path specified to `--split-debug-info`. |
+| `--ios-dysms` | Optional | Upload iOS dSYM files to Datadog |
+| `--ios-dsyms-location` | Optional | Specify the location of iOS dSYM files. By default, these are located at `./build/ios/archive/Runner.xcarchive/dSYMs`. Adding this parameter automatically sets `--ios-dsyms`. |
+| `--android-mapping` | Optional | Upload Android Proguard mapping file. This is usually only necessary if you specify `--obfuscate` as a parameter to `flutter build` |
```

Comment:
```suggestion
| `--android-mapping` | Optional | Upload Android Proguard mapping file. This is only necessary if you specify `--obfuscate` as a parameter to `flutter build`. |
```

Reply:
I think there may be situations where this would be necessary even without the `--obfuscate paraemter`, so I'm going to leave the "usually" in.